CMIP6_studies/Ouranos.yaml

```diff
@@ -0,0 +1,355 @@
+  temporal_scope: Climatology
+  period:
+    reference: 1981-2010
+  plausible_values:
+    min: 0
+    max: +inf  
+    source: Ouranos_SAC
+    comment: >
+    
+  data_source: authors
+  data: 
+   CanESM5-1-EBC: 6.6
+   EC-Earth3-Veg: 12.1 
+   UKESM1-0-LL: 14.2
+   EC-Earth3: 16.0
+   MPI-ESM1-2-HR: 16.2
+   MPI-ESM1-2-LR: 17.3
+   NorESM2-MM: 19.0
+   CanESM5-1: 19.7
+   ACCESS-CM2: 22.0
+   TaiESM1: 26.3
+   NorESM2-LM: 27.0
+   CESM2: 31.2
+   ACCESS-ESM1-5: 33.9
+   CMCC-ESM2: 34.3
+   MIROC6: 36.2
+   CNRM-ESM2-1: 39.4
+   CMCC-CM2-SR5: 39.7
+   CNRM-CM6-1: 50.2
+   IPSL-CM6A-LR: 51.3
+   MIROC-ES2L: 53.0
+   CNRM-CM6-1-HR: 61.8
+
+
+- key: Ouranos zg-500mb filtered variance bias
+  doi: [in-house performance]
+  metric: &zg-500mbvar
+    name: zg500mbvar
+    long_name: STD of band-pass (2–6 days) filtered on daily 500-mb geopotential height climatology vs ERA5 climatology
+    units: m²
+    variables: zg500var
+    comment: >
+      First a band-pass (2-6 days) filtered is applied on the daily 500-mb zg. 
+      Then, we calculate the RMSE at each grid point over the selected the domain (relative to ERA5)
+      for the period from 1981 to 2010 and then we do a spatial average. 
+      All models and ERA5 are linearly interpolated on a regular 1°x1° grid. Metric to analyse bias of the synoptic activity.
+      -----
+      Periods, domains can be changed in https://pavics.ouranos.ca/selection-de-modeles
+    best: 119
+    worst: 601
+  type: performance
+  spatial_scope: NAM-CORDEX
+  temporal_scope: Climatology
+  period:
+    reference: 1981-2010
+  plausible_values:
+    min: 0
+    max: +inf  
+    source: Ouranos_SAC
+    comment: >
+    
+  data_source: authors
+  data: 
+   CESM2: 119
+   NorESM2-MM: 120
+   EC-Earth3-Veg: 132
+   EC-Earth3: 143
+   CanESM5-1: 149
+   UKESM1-0-LL: 150
+   TaiESM1: 165
+   MPI-ESM1-2-HR: 165
+   CNRM-CM6-1-HR: 168
+   CMCC-ESM2: 183
+   ACCESS-ESM1-5: 186
+   MPI-ESM1-2-LR: 187
+   CMCC-CM2-SR5: 188
+   CanESM5-1-EBC: 202
+   NorESM2-LM: 213
+   IPSL-CM6A-LR: 221
+   ACCESS-CM2: 238
+   CNRM-ESM2-1: 260
+   CNRM-CM6-1: 284
+   MIROC6: 344
+   MIROC-ES2L: 601
+
+
+
+- key: Ouranos siconcMaxExt
+  doi: [in-house performance]
+  metric: &siconcMaxExt
+    name: siconcMaxExt
+    long_name: Sea Ice concentration biais for climatology of maximum annual extent of sea ice (models vs ERA5)
+    units: %
+    variables: siconc
+    comment: >
+      First, using monthly data, we find the month with maximum annual sea ice extent.  
+      Then, we calculate the RMSE at each grid point over the selected the domain (relative to ERA5)
+      for the period from 1981 to 2010 and then we do a spatial average. 
+      All models and ERA5 are linearly interpolated on a regular 1°x1° grid. 
+      -----
+      Periods, domains can be changed in https://pavics.ouranos.ca/selection-de-modeles
+    best: 0.033
+    worst: 0.207
+  type: performance
+  spatial_scope: Artic (40-90 latitude)
```

Comment:
The details of the spatial scopes can be provided either as comments in the [spatial scope list for the domain](https://github.com/WCRP-CORDEX/cmip6-for-cordex/blob/e6056efdbb698d17fcbd412db61cd27b43243ca9/CMIP6_studies_config.yaml#L42-L46) and/or in the metric comment. The spatial_scope itself should be just a keyword